vulnerabilities/helpers.py

```diff
@@ -20,16 +20,29 @@
 #  VulnerableCode is a free software from nexB Inc. and others.
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 
+import bisect
+import dataclasses
 import json
 import re
+from typing import Optional
 
 import requests
 import toml
 import urllib3
 import yaml
+from packageurl import PackageURL
+from univers.versions import version_class_by_package_type
 
 # TODO add logging here
 
+is_cve = re.compile(r"CVE-\d{4}-\d{4,7}", re.IGNORECASE).match
+
+
+@dataclasses.dataclass(order=True, frozen=True)
+class AffectedPackageWithPatchedPackage:
```

Comment:
This will be moved somewhere else in another PR

Reply:
Use `AffectedPackageWithPatch`

Reply:
Or just `AffectedPackage` ?